src/components/Restoration/RestorationType.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <v-card flat id="restoration-type" class="pa-6">
+    <v-row>
+
+      <v-col cols="12" sm="3" class="pr-4">
+        <label class="font-weight-bold">Restoration Type</label>
+      </v-col>
+
+      <v-col cols="12" sm="9">
+        <v-radio-group column class="pt-0 mt-0" v-model="selectRestorationType">
+          <v-radio id="full-radio-button" label="Full Restoration" :value=RestorationTypes.FULL />
+
+          <!-- Relationship To Company Checkboxes -->
+          <v-expand-transition>
+            <v-card flat v-if="isFull">
+              <div class="ml-8">
+                Please select
+                <v-tooltip content-class="top-tooltip" transition="fade-transition" top>
+                  <template v-slot:activator="{ on, attrs }">
+                    <span class="dotted-underline" v-bind="attrs" v-on="on">
+                      applicant's relationship
+                    </span>
+                  </template>
+                  Full restoration application must be related to the dissolved business.
+                </v-tooltip>
+                to the company at the time the company was dissolved:
+              </div>
+              <RelationshipsPanel :draftRelationships="getRestoration.relationships"
+                @changed="setRestorationRelationships($event)" @valid="setRestorationTypeValid($event)" />
+            </v-card>
+          </v-expand-transition>
+          <v-radio id="limited-radio-button" label="Limited Restoration" :value=RestorationTypes.LIMITED />
+
+          <!-- Limited Restoration Radio Panel -->
+          <v-expand-transition>
+            <v-card flat v-if="isLimited">
+              <LimitedRestorationPanel :currentDate="getCurrentDate" :expiryDate="getRestoration.expiry"
+                @expiry="setRestorationExpiry($event)" />
+            </v-card>
+          </v-expand-transition>
+
+        </v-radio-group>
+      </v-col>
+    </v-row>
+  </v-card>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Watch, Emit, Prop } from 'vue-property-decorator'
+import { DateMixin, CommonMixin } from '@/mixins'
+import { Getter, Action } from 'vuex-class'
+import { RestorationTypes, RouteNames } from '@/enums'
+import { RelationshipsPanel } from '@bcrs-shared-components/relationships-panel'
+import { LimitedRestorationPanel } from '@bcrs-shared-components/limited-restoration-panel'
+
+@Component({
+  mixins: [
+    DateMixin,
+    CommonMixin
+  ],
+  components: {
+    RelationshipsPanel,
+    LimitedRestorationPanel
+  }
+})
+export default class RestorationType extends Vue {
+  @Getter getRestoration!: RestorationStateIF
+  @Getter getCurrentDate!: string
+  @Getter isRestorationTypeValid!: boolean
+
+  @Action setRestorationType!: ActionBindingIF
+  @Action setRestorationExpiry!: ActionBindingIF
+  @Action setRestorationRelationships!: ActionBindingIF
+  @Action setRestorationTypeValid!: ActionBindingIF
+
+  // Local properties
+  protected isFull = false
```

Comment:
Instead of managing this property, why not use a computed value (getter) instead?
```
get isFull (): boolean {
   return (this.getRestoration.type === RestorationTypes.FULL)
}
```
And ditto for `isLimited`.

Let the framework do the reactivity for you.

Reply:
These two properties are used to determine which panel to expand based on radio button selection. Are you sure having computed value getters would help?

Reply:
I'm 99% sure. Getters are reactive -- they update depending on the properties they look at. And the template is also reactive, so I think your v-cards will conditionally display just fine.